readme.md

```diff
@@ -49,23 +49,25 @@ Select 1:
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
-[] Selects all ul's of class dropdown, only if their children are exclusively li's
+[x] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
 
```

Comment:
This selects all li's where the parent element is a ul with class dropdown. You can review css selectors here: http://www.w3schools.com/cssref/css_selectors.asp